test/radix/README.md

```diff
@@ -0,0 +1,114 @@
+RADIX Test Framework
+====================
+
+Purpose
+-------
+
+This directory contains the RADIX test framework, which is a six-dimension
+script-driven facility intended to facilitate execution of
+
+- multi-stream
+- multi-client
+- multi-server
+- multi-thread
+- multi-process (in future)
+- multi-node (in future)
+
+test vignettes for network protocol testing applications. While it is currently
```

Comment:
question: what are test vignettes? are they kind of set of tests? or is it a different name for a script? or test scenario?

Reply:
Yes &mdash; basically a scenario. An occasionally used term...